graders/java/JUnitAutograder.java

```diff
@@ -59,9 +59,8 @@ public static void main(String args[]) {
 
     public void runTests() throws UngradableException {
 
-        LauncherDiscoveryRequestBuilder requestBuilder = LauncherDiscoveryRequestBuilder.request();
-
         for (String classSrcName : testClasses) {
+	    LauncherDiscoveryRequestBuilder requestBuilder = LauncherDiscoveryRequestBuilder.request();
```

Comment:
Let's avoid mixing tabs and spaces.
```suggestion
        for (String classSrcName : testClasses) {
            LauncherDiscoveryRequestBuilder requestBuilder = LauncherDiscoveryRequestBuilder.request();
```

Reply:
Ah, the caveats of using multiple editors... fixed.